packages/nimble/inst/CppCode/predefinedNimbleLists.cpp

```diff
@@ -246,12 +247,14 @@ void OptimControlNimbleList::copyFromSEXP(SEXP S_nimList_) {
     SET_STRING_ELT(S_pxData, 0, mkChar(".xData"));
     PROTECT(S_trace = findVarInFrame(GET_SLOT(S_nimList_, S_pxData),
```

Comment:
This is entirely generated code, so the actual error is in our compiler. Could you point me towards where in our compiler these lines are being generated?